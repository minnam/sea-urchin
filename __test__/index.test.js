/* eslint-env jest */

import { shallow } from 'enzyme'
import React from 'react'
import renderer from 'react-test-renderer'

import {
  Button,
  Checkbox,
  Checkboxes,
  Dropdown,
  FluidBox,
  Grid,
  InputField,
  NavSelect,
  RadioButton,
  Select,
  Theme,
} from '../src';

describe('Welcome (Snapshot)', () => {
  it('Welcome renders hello world', () => {
    const component = renderer.create(<Checkbox />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
  it('Welcome renders hello world', () => {
    const component = renderer.create(<Checkbox />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
