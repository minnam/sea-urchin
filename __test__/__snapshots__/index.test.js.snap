// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome (Snapshot) Welcome renders hello world 1`] = `
<div
  className="f1sqmtpe f1hykqr2 f2gbr4s f1hykqr2"
  onClick={[Function]}
  onMouseDown={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  onMouseMove={[Function]}
  onMouseOver={[Function]}
  onMouseUp={[Function]}
>
  <div
    className="fifd3s5"
  >
    <div
      className="f1mfn0as"
    />
    <div
      style={
        Object {
          "height": "100%",
          "left": 0,
          "position": "absolute",
          "top": 0,
          "width": "100%",
          "zIndex": 5,
        }
      }
    >
      <div
        style={
          Object {
            "borderBottom": "1px solid rgba(160,160,160,0.5)",
            "height": "100%",
            "position": "absolute",
            "right": 0,
            "transition": "all 0.05s",
            "transitionDelay": "0.15000000000000002s",
            "width": 0,
            "zIndex": 0,
          }
        }
      />
      <div
        style={
          Object {
            "borderLeft": "1px solid rgba(160,160,160,0.5)",
            "bottom": 0,
            "height": 0,
            "left": -1,
            "position": "absolute",
            "transition": "all 0.05s",
            "transitionDelay": "0.1s",
            "width": "100%",
            "zIndex": 0,
          }
        }
      />
      <div
        style={
          Object {
            "borderTop": "1px solid rgba(160,160,160,0.5)",
            "height": "100%",
            "left": 0,
            "position": "absolute",
            "top": -1,
            "transition": "all 0.05s",
            "transitionDelay": "0.05s",
            "width": 0,
            "zIndex": 0,
          }
        }
      />
      <div
        style={
          Object {
            "borderRight": "1px solid rgba(160,160,160,0.5)",
            "height": 0,
            "position": "absolute",
            "right": -1,
            "top": 0,
            "transition": "all 0.05s",
            "transitionDelay": "0s",
            "width": "100%",
            "zIndex": 0,
          }
        }
      />
    </div>
  </div>
  <div
    className="f13pp1e7"
  />
  <div
    style={
      Object {
        "height": "100%",
        "left": 0,
        "position": "absolute",
        "top": 0,
        "width": "100%",
        "zIndex": 5,
      }
    }
  >
    <div
      style={
        Object {
          "borderBottom": "1px solid rgba(160,160,160,0.5)",
          "height": "100%",
          "position": "absolute",
          "right": 0,
          "transition": "all 0.05s",
          "transitionDelay": "0.15000000000000002s",
          "width": 0,
          "zIndex": 0,
        }
      }
    />
    <div
      style={
        Object {
          "borderLeft": "1px solid rgba(160,160,160,0.5)",
          "bottom": 0,
          "height": 0,
          "left": -1,
          "position": "absolute",
          "transition": "all 0.05s",
          "transitionDelay": "0.1s",
          "width": "100%",
          "zIndex": 0,
        }
      }
    />
    <div
      style={
        Object {
          "borderTop": "1px solid rgba(160,160,160,0.5)",
          "height": "100%",
          "left": 0,
          "position": "absolute",
          "top": -1,
          "transition": "all 0.05s",
          "transitionDelay": "0.05s",
          "width": 0,
          "zIndex": 0,
        }
      }
    />
    <div
      style={
        Object {
          "borderRight": "1px solid rgba(160,160,160,0.5)",
          "height": 0,
          "position": "absolute",
          "right": -1,
          "top": 0,
          "transition": "all 0.05s",
          "transitionDelay": "0s",
          "width": "100%",
          "zIndex": 0,
        }
      }
    />
  </div>
</div>
`;
